packages/shell-api/src/field-level-encryption.spec.ts

```diff
@@ -799,7 +799,7 @@ describe('Field Level Encryption', function () {
 
     beforeEach(async function () {
       dbname = `test_fle_${Date.now()}`;
-      uri = `${await testServer.connectionString()}/${dbname}`;
+      uri = await testServer.connectionString();
```

Comment:
```suggestion
      uri = await testServer.connectionString({}, { pathname: `/${dbname}` });
```
Should it have dbname as pathname here, like in other places?

Reply:
We’re not making use of the fact that `dbname` is used here, so I dropped it. No strong feelings, happy to add it back if you prefer. (It’s added in the e2e tests because for the CLI, the db name after the slash indicates the database that the shell is initially connected to.)